lib/note/service.js

```diff
@@ -0,0 +1,108 @@
+
+const { Note, ArchivedNoteAlias, sequelize } = require('../models')
+const realtime = require('../realtime/realtime')
+
+const forbiddenAlias = ['', 'new', 'me', 'history', '403', '404', '500', 'config']
+const sanitize = (alias) => {
+  return alias.replace(/( |\/)/, '')
+}
+
+const asyncGetNote = async (originAliasOrNoteId) => {
```

Comment:
We already have `getNoteById` function in note/index.js

Reply:
Because the `getNoteById` in note/index.js noget pure note row data.
I think `controller` or `service` should be choose the enough data to use.